src/action/auth.ts

```diff
@@ -28,3 +29,64 @@ export async function signIn(formData: FormData) {
     return "Invalid email or password";
   }
 }
+
+
+export async function signUpEmailCheck(formData:FormData){
```

Comment:
`checkSignUpEmail`

Reply:
Look like most of these function should be use on client side function, not the action.
Why client need to ask the server is these email are empty or not?

context: functions that are on the page that have "use server" on first line meant that it will be run on server.